docs/src/content/docs/introduction/op-flow.mdx

```diff
@@ -37,15 +38,15 @@ A client retrieves public details about a payee's Open Payments-enabled account
 
 ## Incoming payment
 
-The client first <LinkOut href='/apis/auth-server/operations/post-request'>requests/receives a grant</LinkOut> from the payee's authorization server to create an `incoming-payment` resource. The client then sends a request to the payee-side resource server to <LinkOut href='/apis/resource-server/operations/create-incoming-payment/'>create the resource</LinkOut>. When created, the resource server returns unique payment details the client will use to address one or more payments to the payee.
+The client first <LinkOut href='/apis/auth-server/operations/post-request'>requests/receives a grant</LinkOut> from the authorization server of the sender's <Tooltip content="account servicing entity" client:load>ASE</Tooltip> to create an `incoming-payment` resource. The client then sends a request to the resource server of the recipient's ASE to <LinkOut href='/apis/resource-server/operations/create-incoming-payment/'>create the resource</LinkOut>. When created, the resource server returns unique payment details the client will use to address one or more payments to the recipient.
```

Comment:
The client first requests/receives a grant from the authorization server of the **recipient's** ASE (not the sender's) to create an incoming-payment resource. 

With that, I think the second sentence can be shortened to:
The client then sends a request to the ASE's resource server to create the resource.